README.md

```diff
@@ -19,41 +19,103 @@ Supported formats:
+| Parameter                          | Description                                                                                                                                                                                          |
+|------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| -i, --inputXmlReport (required)    | The path to Parasoft XML report that you want to transform to SARIF report.                                                                                                                          |
+| -o, --outputSarifReport (Optional) | The path to output SARIF report. If not specified, the SARIF report will be generated in the same directory as input XML report.                                                                     |
+| -t, --toolOrJavaHomeDir (Optional) | The path to Parasoft tools installation or Java home directory. If not specified, transformer will try to use Java from system environment variables.                                                |
+| -p, --projectRootPaths (Optional)  | The absolute path to the project root directory, used to obtain relative file location paths. Use semicolons to separate multiple paths. If not specified, the file location paths will be absolute. |
+
+<br/>If the XML reports generated by Parasoft Jtest, dotTEST, and C/C++test contain sufficient relevant information, the resulting SARIF reports will include files that were scanned during static analysis. For example:
+```code
 "runs" [
   {
     "tool": { ... }
     "results": [ ... ]
     "artifacts": [
-    {
-      "location": {
-        "uri": "file:/C:/workspace/project/src/main/java/Example.java" 
-       } 
+      {
+        "location": {
+            "uri": "file:/C:/workspace/project/example/src/main/java/Example.java" 
+         } 
       },
       {
         "location": {
-          "uri": "file:/C:/workspace/project/src/main/java/Demo.java" 
+            "uri": "file:/C:/workspace/project/demo/src/main/java/Demo.java" 
         }
       }
     ]
   }
 ]
 ```
+For C/C++test Professional 2023.1 or earlier, to include sufficient relevant information in static analysis reports, the reports must be generated with the **Add absolute file paths to XML data** option enabled. You can enable this option on the command line by setting the `report.location_details=true` property in the settings file.
 
-Notes:
-* For C/C++test Professional 2023.1 or earlier, static analysis reports must be generated with the **Add absolute file paths to XML data** option enabled. You can enable this option on the command line by setting the `report.location_details=true` property in the settings file.
+<br/>If a file location path contains the specified project root path, it will become a relative path in the resulting SARIF report. For example: 
```

Comment:
For the two sections to describe the difference for absolute and relative path, I suggest to provide only one example which has example for both cases, and change the description to:
`For a file location path in the SARIF report, if its project root path is specified in -p, its relative path will be shown; otherwise, its absolute path will be shown.`